src/Infrastructure/Controller/Organization/User/DeleteUserController.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Infrastructure\Controller\Organization\User;
+
+use App\Application\CommandBusInterface;
+use App\Application\QueryBusInterface;
+use App\Application\User\Command\DeleteUserOrganizationCommand;
+use App\Application\User\Query\GetOrganizationUserQuery;
+use App\Domain\User\Exception\UserOrganizationNotFoundException;
+use App\Infrastructure\Security\Voter\OrganizationVoter;
+use Symfony\Bundle\SecurityBundle\Security;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException;
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Routing\Requirement\Requirement;
+use Symfony\Component\Routing\RouterInterface;
+use Symfony\Component\Security\Csrf\CsrfToken;
+use Symfony\Component\Security\Csrf\CsrfTokenManagerInterface;
+
+final class DeleteUserController
+{
+    public function __construct(
+        private RouterInterface $router,
+        private CommandBusInterface $commandBus,
+        private QueryBusInterface $queryBus,
+        private Security $security,
+        private CsrfTokenManagerInterface $csrfTokenManager,
+    ) {
+    }
+
+    #[Route(
+        '/organizations/{organizationUuid}/users/{uuid}',
+        name: 'app_users_delete',
+        requirements: ['organizationUuid' => Requirement::UUID, 'uuid' => Requirement::UUID],
+        methods: ['DELETE'],
+    )]
+    public function __invoke(Request $request, string $organizationUuid, string $uuid): RedirectResponse
+    {
+        $csrfToken = new CsrfToken('delete-user', $request->request->get('token'));
+        if (!$this->csrfTokenManager->isTokenValid($csrfToken)) {
+            throw new BadRequestHttpException('Invalid CSRF token');
+        }
+
+        try {
+            $userOrganization = $this->queryBus->handle(new GetOrganizationUserQuery($organizationUuid, $uuid));
```

Comment:
Partout il faut parler de "organization user" et pas l'inverse, vu que c'est le nom de l'entité (`OrganizationUser.php`)

```suggestion
            $organizationUser = $this->queryBus->handle(new GetOrganizationUserQuery($organizationUuid, $uuid));
```